SmartDeviceLink/private/SDLTextAndGraphicUpdateOperation.h

```diff
@@ -22,7 +22,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 typedef void(^SDLTextAndGraphicUpdateCompletionHandler)(NSError *__nullable error);
-typedef void(^CurrentDataUpdatedHandler)(SDLTextAndGraphicState *__nullable newScreenData, NSError *__nullable error);
+typedef void(^CurrentDataUpdatedHandler)(SDLTextAndGraphicState *__nullable newScreenData, NSError *__nullable error, SDLTextAndGraphicState *__nullable errorScreenData);
```

Comment:
I kind of think that instead of passing a separate handler parameter here, we should just package it in the `error.userInfo` dictionary. Key could be something like `@"failedScreenState"`. Then we can get rid of the error data.